backend/core/src/listings/listings.service.spec.ts

```diff
@@ -194,6 +196,56 @@ describe("ListingsService", () => {
       )
     })
 
+    it("should support basic filters with null data", async () => {
```

Comment:
nit: how about "should include listings with missing data if $include_nulls is true"

Reply:
updated